src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -929,8 +929,12 @@ internal static Cci.IGenericParameterReference Translate(TypeParameterSymbol par
             DiagnosticBag diagnostics,
             bool needDeclaration = false)
         {
+            if (fieldSymbol.IsTupleField)
```

Comment:
In the next PR

---
In reply to: [61801140](https://github.com/dotnet/roslyn/pull/11006#discussion_r61801140) [](ancestors = 61801140,61800802)